src/main/java/seedu/address/logic/commands/eventcommands/AddPollOptionCommand.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.logic.commands.eventcommands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
```

Comment:
"M" should go be after "e"?

Reply:
I think it's because event is a folder, Model is the class

Reply:
Ok!